### PR TITLE
Issue #26 - Added remove page functionality to AirViewer

### PR DESCRIPTION
**Pull request by aanuhya31**

### CODE REVIEW

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 1]
```diff
@@ -105,6 +106,12 @@ public class AIRViewerController implements Initializable {
 
 	@FXML
 	private MenuItem deleteAnnotationMenuItem;
+	
+	@FXML
+	VBox rightControls; // Controls on the Rights side of the Scene
+	
+    @FXML
+    Button removePageButton; // Remove the current page in the Document
```
Comment (sireeshanalabothu) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: format the code. lines of the code are not aligned.

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 2]
```diff
@@ -392,6 +400,21 @@ public void handle(ActionEvent e) {
 		refreshUserInterface();
 		return model;
 	}
+	
+	 /**
+     * Initializes the page control for the creation and deletion of
+     */
+
```
Comment (sireeshanalabothu) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java:423]: please organize the comments section well.

**Review by sireeshanalabothu — APPROVED** [review]

**Review by paidikondala — CHANGES_REQUESTED** [review]
I don't see the fxml code for removing pages

**Comment by aanuhya31** [comment]
Added back the removed changes. Please review now and approve changes.

**Review by paidikondala — APPROVED** [review]
Addressed requested changes. Code looks good now.

**Review by aanuhya31 — COMMENTED** [review]
review done and changes looks good hence merged the PR

**Pull request merged**